### PR TITLE
resources page

### PR DESCRIPTION
**Pull request by anushkanema**

### CODE REVIEW

File: public/resources/index.html [hunk 1]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Resources</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
```
Comment (Omkaragrawal) [inline public/resources/index.html:9]: move it to the bottom inside the body tag.

File: public/resources/index.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Resources</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
```
Comment (Omkaragrawal) [inline public/resources/index.html:8]: move it to the bottom inside the body tag.

File: public/resources/index.html [hunk 3]
```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Resources</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```
Comment (Omkaragrawal) [inline public/resources/index.html:7]: move it to the bottom inside the body tag.

**Review by Omkaragrawal — CHANGES_REQUESTED** [review]
move the tags to the bottom inside the body tag.

File: public/resources/index.html (hunk 2 above)
Comment (anushkanema) [reply public/resources/index.html]: Done. Please check updated files

File: public/resources/index.html [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+               <li class="nav-item px-5"><a class="nav-link" href="#">Home</a></li>
+               <li class="nav-item px-5"><a class="nav-link" href="#">Bulletin Board</a></li>
+               <li class="nav-item px-5"><a class="nav-link" href="#">Resources</a></li>
+               <li class="nav-item px-5"><a class="nav-link" href="#">Collaborations</a></li>
+               <li class="nav-item px-5"><a class="nav-link" href="#">About Us</a></li>
+               <li class="nav-item px-5"><a class="nav-link" href="#">Contact Us</a></li>
+               <button type="button" id="logout" class="btn btn-primary">Log Out</button>
+           </ul>
+  </nav>
+<body>
```
Comment (dsc-umit) [inline public/resources/index.html:13]:
```suggestion
  <!DOCTYPE html>
  <html>
  <head>
  <title>Resources</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
  </head>
  <body>
  <nav class="navbar navbar-expand-sm justify-content-center bg-dark navbar-dark">
           <ul class="navbar-nav">
               <li class="nav-item mt-2" id="logo"><h5><b>College Made Easy!</b></h5></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Home</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Bulletin Board</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Resources</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Collaborations</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">About Us</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Contact Us</a></li>
               <button type="button" id="logout" class="btn btn-primary">Log Out</button>
           </ul>
  </nav>
```

File: public/resources/index.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Resources</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+</head>
```
Comment (dsc-umit) [inline public/resources/index.html:91]:
```suggestion
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
```

**Review by dsc-umit — CHANGES_REQUESTED** [review]

File: public/resources/index.html (hunk 4 above)
Comment (Omkaragrawal) [inline public/resources/index.html:13]:
```suggestion
  <!DOCTYPE html>
  <html>
  <head>
  <title>Resources</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
  </head>
  <body>
  <nav class="navbar navbar-expand-sm justify-content-center bg-dark navbar-dark">
           <ul class="navbar-nav">
               <li class="nav-item mt-2" id="logo"><h5><b>College Made Easy!</b></h5></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Home</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Bulletin Board</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Resources</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Collaborations</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">About Us</a></li>
               <li class="nav-item px-5"><a class="nav-link" href="#">Contact Us</a></li>
               <button type="button" id="logout" class="btn btn-primary">Log Out</button>
           </ul>
  </nav>
```

File: public/resources/index.html (hunk 5 above)
Comment (Omkaragrawal) [inline public/resources/index.html:91]:
```suggestion
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
```

**Review by Omkaragrawal — CHANGES_REQUESTED** [review]

**Pull request merged**